readme.md

```diff
@@ -32,7 +32,8 @@ Running
 -------
 Before running unit tests, set PYTEST_QT_API as follows:
 
-    export PYTEST_QT_API='pyqt4v2'
+    *nix: export PYTEST_QT_API='pyqt4v2'
+	Windows: SET PYTEST_QT_API='pyqt4v2'
```

Comment:
Even on windows we recommend using a bash shell, since that's a lot simpler than using cmd.exe along with its associated quirks.